nextcord/core/gateway/protocols/gateway.py

```diff
@@ -51,8 +51,12 @@ def should_reconnect(self, shard: ShardProtocol) -> bool:
     async def close(self) -> None:
         ...
 
-    def _shard_dispatch(self, event_name: str, shard: ShardProtocol, *args: Any) -> None:
+    def _shard_dispatch(
```

Comment:
Unrelater change? Also we use black + isort, please use that over your current linter for ncv3

Reply:
They seem to have used black and isort?

Reply:
> Unrelater change? Also we use black + isort, please use that over your current linter for ncv3

That is as I said when I pulled and linted with black, I guess another previous commit hadn't been linted